councilmatic_core/migrations/0037_unmangle_identifier.py

```diff
@@ -0,0 +1,25 @@
+# Custom migration
+from __future__ import unicode_literals
+import re
+
+from django.db import migrations, models
+
+def unmangle_identifier(apps, schema_editor):
+    Bill = apps.get_model('councilmatic_core', 'Bill')
+    # Bill = apps.get_model('nyc', 'NYCBill')
+
+    _pattern = re.compile(r'([A-Za-z]*\s)(\d{1,3}-\d{4})$')
+
+    for bill in Bill.objects.all():
+        unmangled_identifier = _pattern.sub(lambda match: match.group(1) + ('0' * (9 - len(match.group(2)))) + match.group(2), bill.identifier)
```

Comment:
This is a good opportunity to learn about some more powerful features of python string formatting. In this case, padding: https://pyformat.info/#string_pad_align